src/pyhf/infer/test_statistics.py

```diff
@@ -58,5 +58,5 @@ def qmu(mu, data, pdf, init_pars, par_bounds):
     qmu = fixed_poi_fit_lhood_val - unconstrained_fit_lhood_val
     qmu = tensorlib.where(
         muhatbhat[pdf.config.poi_index] > mu, tensorlib.astensor(0.0), qmu
-    )[0]
+    )
```

Comment:
I'm confused why I'm getting doctest failing here

```
=================================== FAILURES ===================================
___________________ [doctest] pyhf.infer.test_statistics.qmu ___________________
024         >>> pyhf.set_backend("numpy")
025         >>> model = pyhf.simplemodels.hepdata_like(
026         ...     signal_data=[12.0, 11.0], bkg_data=[50.0, 52.0], bkg_uncerts=[3.0, 7.0]
027         ... )
028         >>> observations = [51, 48]
029         >>> data = pyhf.tensorlib.astensor(observations + model.config.auxdata)
030         >>> test_mu = 1.0
031         >>> init_pars = model.config.suggested_init()
032         >>> par_bounds = model.config.suggested_bounds()
033         >>> pyhf.infer.test_statistics.qmu(test_mu, data, model, init_pars, par_bounds)
Expected:
    3.938244920380498
Got:
    array([3.93824492])
```

 as

```python
>>> import pyhf
>>> pyhf.set_backend("numpy")
>>> model = pyhf.simplemodels.hepdata_like(
...     signal_data=[12.0, 11.0], bkg_data=[50.0, 52.0], bkg_uncerts=[3.0, 7.0]
... )
>>> observations = [51, 48]
>>> data = pyhf.tensorlib.astensor(observations + model.config.auxdata)
>>> test_mu = 1.0
>>> init_pars = model.config.suggested_init()
>>> par_bounds = model.config.suggested_bounds()
>>> test_stat = pyhf.infer.test_statistics.qmu(test_mu, data, model, init_pars, par_bounds)
>>> print(f"test_stat: {test_stat} is of type {type(test_stat)}")
test_stat: 3.938244920380498 is of type <class 'numpy.float64'>
```

Reply:
@lukasheinrich @kratsg Can one of you checkout this PR code and see if you can reproduce the above?